README.md

```diff
@@ -14,7 +14,7 @@ provider "newrelic" {
 
 module "newrelic_monitoring" {
   source  = "vistaprint/monitoring/newrelic"
-  version = "2.0.0"
+  version = "3.0.0"
```

Comment:
I suppose we will now publish version 3.0.1. So, I'd use a range (something like `">= 3.0.0, < 4.0.0"`)

Reply:
You forgot to use a version range here.